floyd/client/data.py

```diff
@@ -52,7 +52,9 @@ def create(self, data):
 
             # Add request data
             request_data = []
-            request_data.append(("data", ('data.tar', open(compressed_file_path, 'rb'), 'text/plain')))
+
```

Comment:
Good catch. Should we use [file open context manager here](http://book.pythontips.com/en/latest/context_managers.html)?